src/rpc/mempool.cpp

```diff
@@ -1142,8 +1264,10 @@ void RegisterMempoolRPCCommands(CRPCTable& t)
         {"blockchain", &getmempoolancestors},
         {"blockchain", &getmempooldescendants},
         {"blockchain", &getmempoolentry},
+        {"blockchain", &getmempoolcluster},
         {"blockchain", &gettxspendingprevout},
         {"blockchain", &getmempoolinfo},
+        {"blockchain", &getmempoolfeeratediagram},
```

Comment:
in ca18549ee72: Perhaps set the expectation that there could be interface changes.
```suggestion
        {"hidden", &getmempoolfeeratediagram},
```